infrastructure/running-a-validator/operational-tasks/pause-validating.md

```diff
@@ -1,4 +1,44 @@
 ---
 title: Pause Validating
-description: TODO
----
\ No newline at end of file
+description: Learn how to temporarily pause staking activity in Polkadot using the chill extrinsic, with guidance for validators and nominators.
+---
+
+# Pause Validating
+
+## Introduction
+
+If you need to temporarily stop participating in Polkadot staking activities without fully unbonding your funds, chilling your account allows you to do so efficiently. Chilling removes your node from active validation or nomination in the next era while keeping your funds bonded, making it ideal for planned downtimes or temporary pauses. This guide covers the steps for chilling as a validator or nominator, using the `chill` and `chillOther` extrinsics, and how these affect your staking status and nominations.
+
+## Chilling Your Node
+
+If you need to temporarily step back from staking without unbonding your funds, you can "chill" your account. Chilling pauses your active staking participation, setting your account to inactive in the next era while keeping your funds bonded.
+
+To chill your account, go to the **Network > Staking > Account Actions** page on [Polkadot.js Apps](https://polkadot.js.org/apps){target=\_blank}, and select **Stop**. Alternatively, you can call the [`chill`](https://paritytech.github.io/polkadot-sdk/master/pallet_staking/enum.Call.html#variant.chill){target=\_blank} extrinsic in the `staking` pallet. 
+
+![staking](/images/infrastructure/operational-tasks/staking-keys-stash-proxy.webp)
```

Comment:
This image path should be updated to follow the standards: `file-name-1.webp`. Also, please make sure it's under the `pause-validating` directory

Reply:
Also, the alt text should describe the image a little bit more

Reply:
That image actually had nothing to do with the text so now it's gone.